sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy

```diff
@@ -818,32 +819,6 @@ class BlockBlobAPITest extends APISpec {
         file.delete()
     }
 
-    @Requires({ liveMode() })
-    def "Upload from file no overwrite interrupted"() {
```

Comment:
This test is biggest offender and it doesn't really test client code but how service handles race.
The logical branches present in uploadFromFile are covered in different tests.